.pylintrc

```diff
@@ -0,0 +1,32 @@
+[IMPORTS]
+preferred-modules =
+    unittest:pytest,
+
+[MESSAGES CONTROL]
+
+disable =
+    # On purpose disabled as we rely on black
+    line-too-long,
+    # TODO(ssbarnea): remove temporary skips adding during initial adoption:
```

Comment:
Looks like most of this should be dropped since it's a new project and it's best not to allow weird stuff from the very beginning.